types/react-gauge-chart/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for react-gauge-chart 0.2
+// Project: https://martin36.github.io/react-gauge-chart/
```

Comment:
The link here needs to be github.com/Martin36/react-gauge-chart

Reply:
Done